wagtail_streamfield_migration_toolkit/test/models.py

```diff
@@ -0,0 +1,26 @@
+from statistics import mode
+from django.db import models
+from wagtail.models import Page
+from wagtail.fields import StreamField
+from wagtail.blocks import CharBlock, StreamBlock, StructBlock
+
+
+class StructBlock1(StructBlock):
+    struct1field1 = CharBlock()
+    struct1field2 = CharBlock()
```

Comment:
Making these names more descriptive might make things easier for us, for example `struct1_char1`